script/mailman_server

```diff
@@ -28,6 +28,7 @@ Mailman::Application.run do
 #       retries ||= 0
       # do something with the database that does not fetch mails yet (let's try to ensure we have a MySQL connection first)
       puts Comment.last.inspect
+      Mailman.logger.info "Mail received from #{mail.from.first}: '#{mail.subject}' - '#{mail&.html_part}'"
       Comment.receive_mail(message)
```

Comment:
we can add more after the mail is processed, too, perhaps, to confirm when it's competed. Could we automatically periodically send mail in and filter it with a special subject line, say, once per day? 

```suggestion
      if message.subject == "Incoming mail parsing test"
        Mailman.logger.info "Received incoming mail parsing test at #{Time.now.to_s}"
      end
      last_comment = Comment.last
      Comment.receive_mail(message)
      if Comment.last.id != last_comment.id
        Mailman.logger.info "Comment created from #{mail.from.first}: '#{mail.subject}''"
      end
```